en/subscription_services_tos.md

```diff
@@ -0,0 +1,111 @@
+
+We only offer Mozilla VPN as a paid subscription service.
+
+#### You Must Be Eligible
+You must be in a country where the Service is available to download Mozilla VPN and use the Service. Currently, the Service is available in Austria, Belgium, Canada, Finland, France, Germany, Ireland, Italy, Malaysia, the Netherlands, New Zealand, Singapore, Spain, Sweden, Switzerland, the United Kingdom, and the United States. 
+
+#### Permissions Necessary to Provide Mozilla VPN
+Mozilla VPN is a Virtual Private Network in partnership with Mullvad. To learn more about how the Service works, you can see Mozilla’s source code for the VPN program.
+
+__Your Use of the Service.__ If you have a current, paid subscription to Mozilla VPN, Mozilla gives you permission to use the Mozilla VPN according to these Terms of Service. This permission is only for your personal use. You may not transfer, sublicense, resell, or commercially exploit the service.
+
+You give Mozilla and Mullvad all rights necessary to operate the Service, including processing the data you upload to or access through Mozilla VPN. You can find more information about the data Mozilla and Mullvad process in the Mozilla Subscription Services Privacy Notice.
+
+__Device Limit.__ When your subscription is active, you can connect up to 5 devices to Mozilla VPN.
+
+Your License to Use the Software. The Mozilla VPN software is made available to you under the terms of various open source licenses, including the Mozilla Public License.
+
+## Firefox Relay
+We offer Firefox Relay as either a free service or a paid subscription service. The phone number masking service is a partnership with Twilio. 
+
+#### You Must Be Eligible
+To use the paid subscription service, you must be in a country where Firefox Relay is available. 
+
+* Currently, the phone number masking service is only available in the United States and Canada. 
+* The email masking service is available in Austria, Belgium, Canada, Finland, France, Germany, Ireland, Italy, Malaysia, the Netherlands, New Zealand, Singapore, Spain, Sweden, Switzerland, the United Kingdom, and the United States.
+
+#### Permissions Necessary to Provide Firefox Relay
+__Your Use of Relay.__ If you have a current, paid subscription to Firefox Relay, Mozilla gives you permission to use it according to these Terms of Service. This permission is only for your personal use. You may not transfer, sublicense, resell, or commercially exploit Firefox Relay. You are solely responsible for the Content you transmit and the consequences.
+
+You give Mozilla and Twilio all rights necessary to operate Firefox Relay, including processing your email address and phone number as well as sending and receiving the emails, phone calls, and text messages sent to and from the masks you create.  In the event that an email cannot be delivered to you, we will keep it on our servers and delete it after it has been delivered (in no event will we hold onto it for more than three days). 
```

Comment:
Nit: there shouldn't be double spaces after period, they have no practical effect in markdown or HTML.

Reply:
agreed. I think I got rid of it now.